CRM/Upgrade/Incremental/php/FiveTwentyFive.php

```diff
@@ -79,6 +79,11 @@ public function upgrade_5_25_alpha1($rev) {
     $this->addTask('Convert Report Form dates from jcalander to datepicker', 'convertReportsJcalendarToDatePicker');
   }
 
+  public function upgrade_5_25_beta2($rev) {
```

Comment:
@totten should we bump the version to beta3?    Sites already on beta 2 won't ever hit this

Reply:
Yup, it's better to do as beta3. :+1: Updated/rebased.